browser/resources/settings/brave_overrides/settings_ui.ts

```diff
@@ -91,38 +95,47 @@ const BraveClearSettingsMenuHighlightBehavior = {
     // Clear menu selection after scrolling away.
```

Comment:
to get rid of the `this as SettingsUiElement` casts you might be able to change the type of `this` in the function declaration:


```suggestion
  ready: function(this: SettingsUiElement) {
    // Clear menu selection after scrolling away.
```